aws/elasticache/main.tf

```diff
@@ -85,6 +85,7 @@ resource "aws_security_group" "sg_on_elasticache_instance" {
     from_port       = local.port
     to_port         = local.port
     protocol        = "tcp"
+    cidr_blocks     = var.sg_cidr_blocks
```

Comment:
I think that, as we discussed, and as @rdunlop also discovered, this doesn’t work for Elasticache. Should we remove it from this PR?